docs/src/docs/browsercaps/common-browser-caps.adoc

```diff
@@ -23,19 +23,27 @@ WEB_DRIVER_MANAGER.setGlobalCapability(FirefoxOptions.FIREFOX_OPTIONS, options);
 
 ====
 
+= Request capabilities
 
-== Request capabilities
-
-Some WebDriverRequests support setting capabilities, like the `DesktopWebDriverRequest`.
+A request defined capability means using a Webdriver request like `DesktopWebDriverRequest`.
```

Comment:
"means" seems to be the wrong verb here? Or possibly the object does not fit?
If I turn this around as a test: "Using a Webdriver request [...] means a request defined capability." does not read particularly sensible.

The old text seems more fitting to me, but it presumably was replaced for a reason?

Reply:
Hm... yes... The text is the result of merging the content of deleted `local-caps.adoc`. I reverted the change.